extension/js/common/api/key-server/wkd.ts

```diff
@@ -27,60 +27,47 @@ export class Wkd extends Api {
   }
 
   // returns all the received keys
-  public rawLookupEmail = async (email: string): Promise<{ keys: Key[], errs: Error[] }> => {
+  public rawLookupEmail = async (email: string): Promise<ArmoredKeyIdentityWithEmails[]> => {
     // todo: should we return errs on network failures etc.?
     const parts = email.split('@');
     if (parts.length !== 2) {
-      return { keys: [], errs: [] };
+      return [];
     }
     const [user, recipientDomain] = parts;
     if (!user || !recipientDomain) {
-      return { keys: [], errs: [] };
-    }
-    if (!opgp) {
-      // pgp_block.htm does not have openpgp loaded
-      // the particular usecase (auto-loading pubkeys to verify signatures) is not that important,
-      //    the user typically gets the key loaded from composing anyway
-      // the proper fix would be to run encodeZBase32 through background scripts
-      return { keys: [], errs: [] };
+      return [];
     }
     const lowerCaseRecipientDomain = recipientDomain.toLowerCase();
-    const directDomain = lowerCaseRecipientDomain;
     const timeout = (this.usesKeyManager && lowerCaseRecipientDomain === this.domainName) ? 10 : 4;
-    const advancedDomainPrefix = (directDomain === 'localhost') ? '' : 'openpgpkey.';
-    const hu = opgp.util.encodeZBase32(await opgp.crypto.hash.digest(opgp.enums.hash.sha1, Buf.fromUtfStr(user.toLowerCase())));
-    const directHost = (typeof this.port === 'undefined') ? directDomain : `${directDomain}:${this.port}`;
-    const advancedHost = `${advancedDomainPrefix}${directHost}`;
+    const hashed = await window.crypto.subtle.digest('SHA-1', Buf.fromUtfStr(user.toLowerCase()));
+    const hu = this.encodeZBase32(new Uint8Array(hashed));
+    const directHost = WKD_API_HOST || `https://${lowerCaseRecipientDomain}`; // lgtm [js/trivial-conditional]
+    const advancedHost = WKD_API_HOST || `https://openpgpkey.${lowerCaseRecipientDomain}`; // lgtm [js/trivial-conditional]
```

Comment:
we can add subdomains of `localhost` -- not a problem --  I tried that with `gmail.localhost` -- it works perfectly well.
Maybe we don't even need to set up fes domain in `hosts` and use `fes.localhost`?

Reply:
Unless you want to explicitly specify `https://` in `WKD_API_HOST`...

Reply:
If I didn't know that! That means we can throw away the requirement to set up domains with a hosts file. Let's look into this in another issue, it will be a larger change to the tests.

Reply:
#4631